source/docs/ROS/getting-started.rst

```diff
@@ -0,0 +1,67 @@
+Getting Started
+===============
+
+Downloading the Base Repo
+-------------------------
+
+To get started make sure ROS is installed on your machine:
+
+.. note:: The following instructions can be found in the Studica ROS repository on github: ``https://github.com/studica/ros-dev``.
+
+Installing ROS
+^^^^^^^^^^^^^^
+
+1. If you haven't already, run ``installNoetic.sh`` to install ROS Noetic. For more information, visit http://wiki.ros.org/noetic/Installation/Debian
+
+2. Download catkin tools from https://catkin-tools.readthedocs.io/en/latest/installing.html, then run ``sudo apt install python3-catkin-tools python3-osrf-pycommon`` to finalize the install.
+
+.. attention:: Running the ``installNoetic.sh`` script will require some time, this process can take approximately 3 hours. 
+
+Creating a ROS Workspace
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Create a directory ``catkin_ws`` by running ``mkdir -p ~/catkin_ws/src``, preferrably in ``/home/pi``. Then ``cd catkin_ws`` and run ``catkin init`` to initialize the workspace environment.
```

Comment:
Should switch to code blocks as it makes it easier to copy and paste

Reply:
Even though it's just a series of commands in the terminal?

Reply:
Wait nevermind